internal/pkg/deploy/cloudformation/stack/transformers.go

```diff
@@ -77,11 +83,23 @@ func convertSidecar(s map[string]*manifest.SidecarConfig) ([]*template.SidecarOp
 			Variables:    config.Variables,
 			MountPoints:  mp,
 			DockerLabels: config.DockerLabels,
+			DependsOn:    config.DependsOn,
 		})
 	}
 	return sidecars, nil
 }
 
+// convertDependsOn converts an Image DependsOn field to a template DependsOn version
+func convertDependsOn(i *manifest.Image, s map[string]*manifest.SidecarConfig, m string) (map[string]string, error) {
```

Comment:
This function isn't being called anywhere (yet), right? It'll go in `convertSidecar()`?

Reply:
This is the conversion for the DependsOn field from the manifest.image, so it would go into the jobs. Should I rename it to convertImageDependsOn?

Reply:
I currently have adding the image DependsOn saved for when I do integration tests.  Should it be added now?

Reply:
Ah, sorry, yes-- I was getting the two Dependsons confused! Probably just me, but renaming for clarity wouldn't hurt! :)

Reply:
>

> I currently have adding the image DependsOn saved for when I do integration tests. Should it be added to the jobs now?

You can add it in the next PR!

Reply:
Clarified naming to convertImageDependsOn